README.md

```diff
@@ -24,3 +24,21 @@ translators in seconds.
 Zanata is Free software, licensed under the [LGPL][].
 
 [LGPL]: http://www.gnu.org/licenses/lgpl-2.1.html
+
+Developers
+----------
+
+You can use one of the scripts to build the project using maven:
+
+[`etc/scripts/quickbuild.sh`](etc/scripts/quickbuild.sh) - Builds the project as quickly as possible, targeting all
+browsers in the GWT components, and skipping all checks and verifications (i.e.
```

Comment:
> It's the quickest we can get to building a fully-functioning Zanata archive.

But what is the purpose of that archive? No-one should use it for production, especially since you skipped all the tests and jdk compatibility checks.

Working for one or two browsers should be enough for most purposes. Ah! What about defaulting to -Dchromefirefox?